Tools/mapGlobe.py

```diff
@@ -27,21 +27,20 @@
 ##@param[in]   labx
 ##@retval      Pred_Y_map             predicted map of target variables, masking nan pixels
 ##@retval      Pred_Y                 predicted map of target variables, without masking
-def extrp_global(packdata, ipft, PFTmask, XVarName, Tree_Ens, colum, Nm, labx):
+def extrp_global(packdata, PFTmask, XVarName, Tree_Ens, colum, Nm, labx):
     global_X_map = np.full((len(XVarName), packdata.nlat, packdata.nlon), np.nan)
     # PFTmask[np.isnan(PFTmask)]=0
-    pmask = np.squeeze(PFTmask[ipft - 1][:])
     Pred_Y = np.full(PFTmask[0].shape, np.nan)
     # global metrics -> dataframe
     for ii in range(len(XVarName)):
         if ii < packdata.Nv_nopft:
```

Comment:
The code currently fails on this line as `packdata.Nv_nopft` has been removed from `main.py`. This was originally defined to be the `len(var_pred_name1` which corresponds to `varlist["pred"]["allname"]` which itself is ` "allname" : ["Tair", "Rainf", "Snowf", "Qair", "PSurf", "SWdown", "LWdown", "clay_frac",
             "interx"],`

Reply:
At this point I am quite confused. 

We encounter variables `NPP0` and `LAI0`, which are Net Primary Productivity and Leaf Area Index. They both have dimensions: `veget, lat, lon`, according to packdata. 


`packdata[XVarName[ii]` has dimensio